apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -946,8 +952,25 @@ public partial class BigtableClientImpl
             ApplyResourcePrefixHeader(ref settings, request.TableName);
         }
 
-        partial void Modify_MutateRowsRequest(ref MutateRowsRequest request, ref CallSettings settings) =>
+        partial void Modify_MutateRowsRequest(ref MutateRowsRequest request, ref CallSettings settings)
+        {
+            // Strip off the retry settings specified. We will apply them to the response stream.
+            // However, keep the overall expiration so the first individual call can use it.
+            if (settings?.Timing?.Retry != null)
+            {
+                settings = settings.WithCallTiming(CallTiming.FromExpiration(settings.Timing.Retry.TotalExpiration));
+            }
```

Comment:
Actually, I think I just realized an error with this approach. The settings passed in here will probably not have the defaults from the BigtableSettings merged in, so I don't think we'd get the RetrySettings here by default. I'll verify tomorrow and fix it if it is an issue. Perhaps I can add some testing into the integration tests as well to verify the type of stream returned that it is setup correctly. I think some logic here and in the method below might change, but feel free to review the rest of the PR.